### DISCUSSION TITLE
What does "Note: Discussion category forms are currently in limited public beta and subject to change." mean?

### DISCUSSION THREAD

**Discussion by justinbarry** [community/community#47838] [Discussions] [answered]
### Select Topic Area

Question

### Body

Do I need to opt-into Discussion category forms? If so, how do I go about that?

**Comment by LindseyB** [comment, marked as answer]
What the disclaimer means is that some of the functionality may change unexpectedly, but you do not need to opt into them you can just use them now.

https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms
